### PR TITLE
Updates to reflect the 3.4.0 release

### PR DESCRIPTION
**Pull request by atomb**

### CODE REVIEW

File: docs/DafnyRef/Types.md [hunk 1]
```diff
@@ -1274,7 +1274,7 @@ type.  For such assignments, an explicit conversion must be used, see
 
 Dafny builds in three families of subset types, as described next.
 
-### 11.3.1 Type`nat`
+### 11.3.1.1 Type`nat`
```
Comment (cpitclaudel) [inline docs/DafnyRef/Types.md:1277]: ?

Comment (atomb) [reply docs/DafnyRef/Types.md:1277]: I think this was done by one of the packaging scripts... Looking at it now, I don't think it should be included.

Comment (cpitclaudel) [reply docs/DafnyRef/Types.md]: There's some fancy regex magic going on somewhere :)

**Comment by cpitclaudel** [comment]
Do I understand correctly that this test always runs on the release?  Does it only run only when we make a new release, or with every PR?

**Comment by atomb** [comment]
> Do I understand correctly that this test always runs on the release? Does it only run only when we make a new release, or with every PR?

It looks like it runs only when we make a new release or when it's manually triggered.

**Comment by atomb** [comment]
Oh, but this PR is obsolete now that @fabiomadge has released 3.4.1. I'm going to close it.

**Pull request closed without merging**